akka-stream-testkit/build.sbt

```diff
@@ -1,8 +1,6 @@
 import akka._
-import com.typesafe.tools.mima.plugin.MimaKeys
 
 AkkaBuild.defaultSettings
 Formatting.formatSettings
 OSGi.streamTestkit
 Dependencies.streamTestkit
```

Comment:
testkit AFAIR also does not guarantee BC, so could have plugin disabled here right?